recipes/CondaNSIS/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "condansis" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0e44381750b27041e1976f26ac04cb66503a933970d8c52208ac3fa43d8551a3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```

Comment:
this should be host, not build

Reply:
Thanks for the review! It's now been changed to host.

Also, the osx test is failing due to a missing dependency.